docker/Dockerfile-api

```diff
@@ -7,33 +7,24 @@ RUN dnf -y install \
     --setopt=install_weak_deps=false \
     --setopt=tsflags=nodocs \
     httpd \
+    httpd-devel \
+    gcc \
     mod_auth_gssapi \
     mod_ssl \
-    python3-celery \
-    python3-flask \
-    python3-flask-login \
-    python3-flask-migrate \
-    python3-flask-sqlalchemy \
-    python3-GitPython \
-    python3-mod_wsgi \
-    python3-packaging \
+    krb5-devel \
     python3-pip \
-    python3-psycopg2 \
-    python3-requests \
-    python3-requests-kerberos \
-    python3-semver \
     python3-setuptools \
+    python-devel \
     && dnf clean all
 COPY . .
+COPY ./docker/cachito-httpd.conf /etc/httpd/conf/httpd.conf
 
-# FIXME: there was a regression in py-amqp. It was supposed to be fixed in
-# https://github.com/celery/py-amqp/pull/350
-# but cachito is still being affected by the regression
-RUN rpm -e --nodeps python3-amqp && pip3 install amqp==5.0.2
+RUN pip3 install -r requirements.txt --no-deps --require-hashes
+RUN pip3 install -r requirements-web.txt --no-deps --require-hashes
+RUN pip3 install . --no-deps
+
+# Silent gitpython dependency because cachito-api doesn't use git
```

Comment:
```suggestion
# Disable gitpython check for the git executable, cachito-api doesn't use git
```

Reply:
The question that one might rightfully ask is: "Why does cachito-api install gitpython at all then?"

The answer is that we unfortunately do not properly separate API deps and worker deps. Worker deps are all normal dependencies, API deps are all normal dependencies + extra `web` dependencies.

Reply:
Yes, do you think we need to add a comment in `requirements.in` ?

Reply:
What kind of comment do you have in mind?

Reply:
Something about the fact that `requirements.*` keeps dependencies for cachito-api and cachito-workers. On other hand, it's pretty obvious

Reply:
Yeah, I think it's not needed